ecosystem/typescript/sdk/examples/typescript/multisig_transaction.ts

```diff
@@ -0,0 +1,121 @@
+
+  // TS SDK support 3 types of transaction payloads: `ScriptFunction`, `Script` and `Module`.
+  // See https://aptos-labs.github.io/ts-sdk-doc/ for the details.
+  const scriptFunctionPayload = new TxnBuilderTypes.TransactionPayloadScriptFunction(
+    TxnBuilderTypes.ScriptFunction.natual(
+      // Fully qualified module name, `AccountAddress::ModuleName`
+      '0x1::Coin',
+      // Module function
+      'transfer',
+      // The coin type to transfer
+      [token],
+      // Arguments for function `transfer`: receiver account address and amount to transfer
+      [BCS.bcsToBytes(TxnBuilderTypes.AccountAddress.fromHex(account4.address())), BCS.bcsSerializeUint64(123)],
+    ),
+  );
+
+  const [{ sequence_number: sequnceNumber }, chainId] = await Promise.all([
+    client.getAccount(mutisigAccountAddress),
+    client.getChainId(),
+  ]);
+
+  // See class definiton here
+  // https://aptos-labs.github.io/ts-sdk-doc/classes/TxnBuilderTypes.RawTransaction.html#constructor.
+  const rawTxn = new TxnBuilderTypes.RawTransaction(
+    // Transaction sender account address
+    TxnBuilderTypes.AccountAddress.fromHex(mutisigAccountAddress),
+    BigInt(sequnceNumber),
+    scriptFunctionPayload,
+    // Max gas unit to spend
+    1000n,
+    // Gas price per unit
+    1n,
+    // Expiration timestamp. Transaction is discarded if it is not executed within 10 seconds from now.
+    BigInt(Math.floor(Date.now() / 1000) + 10),
+    new TxnBuilderTypes.ChainId(chainId),
+  );
+
+  // account1 and account3 sign the transaction
+  const txnBuilder = new TransactionBuilderMultiEd25519((signingMessage: SigningMessage) => {
+    const sigHexStr1 = account1.signBuffer(signingMessage);
+    const sigHexStr3 = account3.signBuffer(signingMessage);
+
+    // Bitmap masks which public key has signed transaction.
+    // See https://aptos-labs.github.io/ts-sdk-doc/classes/TxnBuilderTypes.MultiEd25519Signature.html#createBitmap
+    const bitmap = TxnBuilderTypes.MultiEd25519Signature.createBitmap([0, 2]);
+
+    // See https://aptos-labs.github.io/ts-sdk-doc/classes/TxnBuilderTypes.MultiEd25519Signature.html#constructor
+    const muliEd25519Sig = new TxnBuilderTypes.MultiEd25519Signature(
+      [
+        new TxnBuilderTypes.Ed25519Signature(sigHexStr1.toUint8Array()),
+        new TxnBuilderTypes.Ed25519Signature(sigHexStr3.toUint8Array()),
+      ],
+      bitmap,
+    );
+
+    return muliEd25519Sig;
+  }, multiSigPublicKey);
+
+  const bcsTxn = txnBuilder.sign(rawTxn);
+  const transactionRes = await client.submitSignedBCSTransaction(bcsTxn);
+
+  await client.waitForTransaction(transactionRes.hash);
+
+  resources = await client.getAccountResources(account4.address());
+  accountResource = resources.find((r) => r.type === '0x1::Coin::CoinStore<0x1::TestCoin::TestCoin>');
+  console.log(`multisig account coins: ${(accountResource.data as any).coin.value}. Should be 123!`);
```

Comment:
we should use some sort of assert here so that we can actually make these tests too.